src/puppetlabs/puppetdb/http/aggregate_event_counts.clj

```diff
@@ -5,15 +5,16 @@
 
 (defn routes
   [version optional-handlers]
-  (let [param-spec {:required ["summarize_by"]
+  (let [handlers (cons (partial http-q/restrict-query-to-entity "aggregate_event_counts")
```

Comment:
endpoints need to wrap their queries in a from as you can see here now, since the from is expected in the query when using the query engine functions.